src/services-using.rst

```diff
@@ -32,29 +32,36 @@ Before you are able to synchronize your data or application files, you will need
 
 Authenticating Users
 ----------------------
-To log in or change the authenticated user, launch Services and open the user authentication screen. There are two ways to do this:
+To log in or change the authenticated user, launch Services and open the user verification screen:
 
-  - **Launching From the Home-Screen**: Press the Action Button (:guilabel:`⋮`) and select :menuselection:`Change User/Logout`
+  1. From the Home-Screen, click the :guilabel:`SIGN IN` button.
 
-    .. image:: /img/services-using/services-options-change.*
-      :alt: Services Change User Option
+    .. image:: /img/services-using/services-sign-in.*
+      :alt: Services Sign In Button
       :class: device-screen-vertical
 
-  - **Launching From the Sync Screen**: From within the Sync screen, press the :guilabel:`Change User` button.
+  2. On the next screen select :guilabel:`AUTHENTICATED USER`.
 
-
-    .. image:: /img/services-using/services-sync.*
-      :alt: Sync Screen
+    .. image:: /img/services-using/services-authenticated-user-sign-in.*
+      :alt: Services Authenticate Credentials Verification
       :class: device-screen-vertical
 
-You will see the user authentication screen.
+  3. Next, input the appropriate credentials in the :guilabel:`Username` and :guilabel:`Password` fields and click the :guilabel:`SIGN-IN` button. This will contact the Cloud Endpoint and log in as the new user.
```

Comment:
"new user" might be confusing here. Maybe just end at log in or simply log in as the user.

Reply:
Done